---
title: "'Finally' ne peut pas figurer en dehors d’une instruction 'Try'."
ms.date: 07/20/2015
f1_keywords:
- vbc30382
- bc30382
helpviewer_keywords:
- BC30382
ms.assetid: 0314d8d2-18bc-4bbd-858c-0a18408b52fd
ms.openlocfilehash: 1d5987d6dc3b25088a55b6f0f43a8fb689f2c3e0
ms.sourcegitcommit: 0be8a279af6d8a43e03141e349d3efd5d35f8767
ms.translationtype: HT
ms.contentlocale: fr-FR
ms.lasthandoff: 04/18/2019
ms.locfileid: "59306350"
---
# <a name="finally-cannot-appear-outside-a-try-statement"></a>'Finally' ne peut pas figurer en dehors d’une instruction 'Try'.
L’instruction`Finally` permet de terminer un bloc `Try...Catch...Finally` ; elle ne peut donc figurer qu’une seule fois à la fin du bloc. Soit vous avez une instruction `Finally`inutile, soit votre instruction `Finally` apparaît en dehors des limites de son bloc `Try` correspondant.  
  
 **ID d’erreur :** BC30382  
  
## <a name="to-correct-this-error"></a>Pour corriger cette erreur  
  
1. Recherchez et supprimez l’instruction `Finally` inutile.  
  
2. Déplacez l’instruction `Finally` vers l’emplacement approprié dans votre code.  
  
## <a name="see-also"></a>Voir aussi

- [Try...Catch...Finally (instruction)](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)
